<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(20, 29, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="windowTabs">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(27, 40, 54);
color: rgb(255, 255, 255);
background-color: rgb(20, 29, 38);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>20</height>
       </size>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mixerTab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../tststst/dsp khara/task2/nyquist-theory-illustrator/data/icons/Calculator-256.png</normaloff>../../../../tststst/dsp khara/task2/nyquist-theory-illustrator/data/icons/Calculator-256.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Mixer</string>
       </attribute>
       <widget class="PlotWidget" name="mixerGraph1">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>721</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="magnitudeMixerSlider">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>320</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="freqMixerSlider">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>380</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="phaseMixerSlider">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>430</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>320</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Magnitude</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>380</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>430</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phase</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="magnitudeMixerLcd">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>320</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="freqMixerLcd">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>380</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="phaseMixerLcd">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>440</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmMixerButton">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>510</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteMixerButton">
        <property name="geometry">
         <rect>
          <x>1040</x>
          <y>510</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QComboBox" name="signalMixerMenu">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>280</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sampleMixerButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>500</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearMixerButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>500</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="viewerTab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../tststst/dsp khara/task2/nyquist-theory-illustrator/data/icons/Real-Time-256.png</normaloff>../../../../tststst/dsp khara/task2/nyquist-theory-illustrator/data/icons/Real-Time-256.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Sampler</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="PlotWidget" name="samplerGraph1"/>
          </item>
          <item>
           <widget class="PlotWidget" name="samplerGraph2"/>
          </item>
          <item>
           <widget class="PlotWidget" name="samplerGraph3"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Max Frequency</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="maxFreqSamplerLcd"/>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Hz</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Sampling Frequency</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="FreqSamplerSlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>30</horstretch>
                  <verstretch>49</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLCDNumber" name="freqSamplerLcd"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Hz</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLCDNumber" name="freqSamplerFactorLcd"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>x Fmax</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="browseSamplerButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="clearSamplerButton">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="splitSamplerButton">
                <property name="text">
                 <string>Hide Graph3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
